pages/27.Mautic 3 upgrade/01.getting-started/docs.en.md

```diff
@@ -13,8 +13,38 @@ That's absolutely fine! This is handy, for example, when you use custom plugins
 
 Then, clear your cache. The Mautic 3 upgrade notification will now disappear. If it doesn't disappear immediately, you can run `app/console mautic:update:find` on the command line so that it's hidden immediately.
 
+### Check the minimum requirements
+
+Mautic 3 has some higher minimum requirements than Mautic 2, and the upgrade has some built in checks to ensure that these are met, in addition to ensuring you have the resources available in your hosting environment to complete the upgrade process. Check the current requirements for the latest version of Mautic at [mautic.org/download/requirements][requirements].  
+
+The main things to be aware of are:
+
+1. PHP must be at 7.3 as a minimum, but ideally 7.4. Check this in the system information for your Mautic instance, or using the command `php -v`at the command line - note that sometimes a different version of PHP will be used at the command line than what is is use for your Mautic instance.
```

Comment:
Sorry @RCheesley but this is partly incorrect - for the upgrade itself the highest possible version is 7.3, as it needs to be a PHP version that both Mautic 2 and 3 have "in common". Otherwise the PHP version will be too new for the upgrade to start 🙈 details in https://github.com/mautic/mautic/blob/2.16/upgrade_v3.php#L58-L59

Reply:
OK - so we should say that the highest version should be 7.3 and that you can update to 7.4 once you're on Mautic 3? If so will need a new PR as this one is merged already.

Reply:
Yes that's correct 👍🏼